decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -38,8 +51,8 @@ namespace :decidim do
       system! "npm uninstall #{webpacker_packages.join(" ")}"
 
       # Modify the webpack binstubs
-      add_binstub_load_path "bin/webpack"
-      add_binstub_load_path "bin/webpack-dev-server"
+      add_binstub_load_path "bin/webpacker"
+      add_binstub_load_path "bin/webpacker-dev-server"
```

Comment:
Note that you need to do this change also in here for it to work with the bin script:
https://github.com/decidim/decidim/blob/80009fc449689b33b4c60efaa7ac1a6c86e556f4/decidim-core/lib/tasks/procfile.rake#L11

At the same time, could you also rename that rake file to `decidim_procfile.rake` to make it consistent with the other rake files there. Sorry, didn't notice this at #10519.

Also, I had some issue when I tried the procfile installation together with the webpacker branch. After running that command I tried to run `./bin/dev` and I received this error:

```
./bin/dev 
sh: 0: cannot open ./bin/dev: Permission denied
```

Which is weird because we have the `chmod` statement here:
https://github.com/decidim/decidim/blob/80009fc449689b33b4c60efaa7ac1a6c86e556f4/decidim-core/lib/tasks/procfile.rake#L23

But I would double check that it is actually working in a fresh development_app because for me it wasn't working and I had to do the `chmod` manually.

Reply:
This issue is that this chmod is incorrect:
https://github.com/decidim/decidim/blob/9e6c5efead87dd268ff7231d424e58e80577b623/decidim-core/lib/tasks/procfile.rake#L23

It gives the following permissions for that file (write and execute but not read):
```
--wxrw--wt
```

It should be:
```ruby
      actions :chmod, "bin/dev", 0o755
```